### ISSUE TITLE
Invalid border width on windows on rotated monitors

### ISSUE THREAD

**Issue by Icelk** [hyprwm/Hyprland#1111]
# Steps to reproduce

I'm using the latest git commit.

Set a monitor to be rotate (e.g. 90°):
```
monitor=DP-2,transform,1
```
Add a border width:
```
general {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more

    gaps_in = 6
    gaps_out = 10
    border_size = 2
    col.active_border = rgba(e8eff2bf)
    col.inactive_border = rgba(e8eff27f)

    layout = dwindle
}
```

Not layout, drop_shadow, rounding, nor blur fixes this.

Open a window. It needs to be rectangular and not close to a square to not work.

# Expected outcome

A border with consistent size

# Noted outcome

Extremely over-sized border on the short sides. The border only becomes larger the more stretched out the window is.

![screenshot-2022-11-26-20:19:58](https://user-images.githubusercontent.com/62830417/204105824-4abcf5fe-d89f-45b2-a76b-16dc46eae6e7.png)

**Comment by vaxerski** [comment]
fixed in 0db75852f326de0433c880d010dd6427be193199

**Comment by Icelk** [comment]
Thanks a lot!
